docs/packages/package_main.rst

```diff
@@ -0,0 +1,65 @@
+.. _Mesa-Packages: 
+
+Mesa Packages 
+==============
+
+*Supporting MESA Agent Based Modeling with Crowd Sourced Packages*  
+------------------------------------------------------------------
+
+	*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
+	This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
+	complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
+	(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
+
+	Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
+	(Nov. 11, 2005), pp. 987-991
+
+
+Mesa Packages Purpose: 
+----------------------
+
+
+**MESA Packages is a repository to share packages for use in Agent Based Models (ABMs) in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."**
+
+	**(1) Provide code which can be integrated into MESA ABMs to enable greater exploration of complex phenomenon.**
+	
+	**(2) Support new programmers by providing detailed guides on "How to" build and share a package.**
+
+
+Mesa Packages Overview: 
```

Comment:
I feel like I need to stop the review here pending updates. This is because some of what is written is written for a collective repository of packages, which is not what we are encouraging. We are encouraging people to write their own packages under their own github usernames and then make the community award of them, which can be done via the list serv and maybe if they notify us, we can add it to our wiki page until one day when create a directory of packages that are floating out in the world. 

^^ Can you update the docs to send that message to folks?